Content.Client/Shuttles/UI/RadarControl.cs

```diff
@@ -16,6 +16,7 @@
 using Robust.Shared.Physics.Collision.Shapes;
 using Robust.Shared.Physics.Components;
 using Robust.Shared.Utility;
+using Content.Shared.MassMedia.Components;
```

Comment:
is this used anywhere?

Reply:
No idea. I don't recall touching that. Might have been one of those errors I clicked the autofix button on, and I believe it only changed capitalization and unnecessary parenthesis.

Reply:
Should I reconfigure my code, to make sure nothing else was changed by mistake?

Reply:
sometimes your IDE will auto-insert dependencies while you are working on code that go unused. I only say it because it appears that you dont use this dependency at all

Reply:
Should I do something about this?

Reply:
if the dependency is unused, it should be removed to keep the code base cleaned up.